install/bootstrap_livecd_usb.md

```diff
@@ -693,6 +693,8 @@ On first log in (over SSH or at local console), the LiveCD will prompt the admin
 
    ```bash
    pit# script -af /var/www/ephemeral/prep/admin/booted-csm-livecd.$(date +%Y-%m-%d).txt
+   pit# PITDATA=$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/PITDATA)
```

Comment:
Why are we setting this variable here if it was already set in /etc/environment?
And the assignment here will only persist for this shell session -- any future shell sessions (as well as any subprocesses of this process) will use the value from the environment file.

Reply:
I actually didn't notice the usage of `/etc/environment` here on Friday, thank you for pointing that out.